src/desktop/apps/authentication/helpers.ts

```diff
@@ -112,19 +119,55 @@ export const setCookies = options => {
   }
 }
 
-export const getRedirect = type => {
+export async function apiAuthWithRedirectUrl(
+  response: any,
```

Comment:
This can be properly typed if needed. `import { Response } from "express"`

Reply:
Resolved by 30bf40e. Thanks for the pointer!